static/markdown/a-confluence-of-waters.md

```diff
@@ -0,0 +1,76 @@
+![](https://two.compost.digital/images/title-images/bluemarbles-700x350.jpg)
```

Comment:
The Distributed Press side should return metadata and signatures for these. But we won't do that in this issue.